inception-project/inception
# #4576 - Allow setting a document-level annotation when bulk processing

**What's in the PR**
- Added dropdown to select document metadata layer and option to fill in values
- Automatically add this to the processed documents

**How to test manually**
* Create a document-level annotation layer
* Go to the process page
* Select the layer
* Fill in featuers
* Run the process
* Check if all processed files have the document-level annotation

**Automatic testing**

**Documentation**
